### ISSUE TITLE
View password popup [feature request]

### ISSUE THREAD

**Issue by fabiopovoledo** [nuxsmin/sysPass#727]
Hello, would it be possibile to provide for the "view password popup" (the window that appear when clicking the opened locker on the account view) to show also any other custom field set as "password" type? In some case it happens that an account can require both a password and a PIN (eg. bank accounts) and it would be great to see both in the same popup.

Thanks in advance and thanks also  for the great work you have done so far

fabio

**Comment by vmario89** [comment]
> 
> 
> Hello, would it be possibile to provide for the "view password popup" (the window that appear when clicking the opened locker on the account view) to show also any other custom field set as "password" type? In some case it happens that an account can require both a password and a PIN (eg. bank accounts) and it would be great to see both in the same popup.
> 
> Thanks in advance and thanks also for the great work you have done so far
> 
> fabio

good idea! Some customized popup view belonging to some predefined account types/masks (could be tied up with presets/templates).

another usecase example from myself could be an SSL cert:
show the account title, the private key, the cert and the ca bundle - (the password would be the password to decrypt the private key in this case)

another example would be some account which needs password and a one time password (TOTP)
